Make.config

```diff
@@ -306,7 +306,7 @@ MIN_WATCHOS_SIMULATOR_VERSION=8.0
 MIN_WATCHOS_COMPANION_SIMULATOR_VERSION=15.0
 MIN_TVOS_SIMULATOR_VERSION=15.0
 # These are the simulator package ids for the versions above
-EXTRA_SIMULATORS=com.apple.pkg.AppleTVSimulatorSDK15_0 com.apple.pkg.iPhoneSimulatorSDK15_0 com.apple.pkg.WatchSimulatorSDK8_0
+EXTRA_SIMULATORS=com.apple.pkg.iPhoneSimulatorSDK15_0 com.apple.pkg.AppleTVSimulatorSDK15_0 com.apple.pkg.WatchSimulatorSDK8_0
```

Comment:
Is the order here significant?

Reply:
should not unless we have noticed something weird